packages/web-components/src/components/rux-pop-up/rux-pop-up.tsx

```diff
@@ -133,6 +140,10 @@ export class RuxPopUp {
         this._handleTriggerKeyPress = this._handleTriggerKeyPress.bind(this)
         this._handleOutsideClick = this._handleOutsideClick.bind(this)
         this._setTriggerTabIndex = this._setTriggerTabIndex.bind(this)
+        if (this.open && !this._assignedOutsideClickEvent) {
+            window.addEventListener('mousedown', this._handleOutsideClick)
+            this._assignedOutsideClickEvent = true
+        }
```

Comment:
@FMorrison87 adding this to the connectedCallback is how I got around the issue of the pop-up not closing on an off click if it was opened by default.